bin/bootstrap.bash

```diff
@@ -8,20 +8,12 @@ set -eu -o pipefail
 here="$(dirname "$(readlink -f "$0")")"
 # shellcheck disable=SC1090
 source "${here}/common.bash"
-: "${config[infrastructure_file]:?Missing infrastructure file}"
 : "${secrets[kube_config_sc]:?Missing service cluster kubeconfig}"
 : "${secrets[kube_config_wc]:?Missing workload cluster kubeconfig}"
 export bootstrap_path="${here}/../bootstrap"
 export scripts_path
 export common_path="${here}/common.bash"
 
-bootstrap_init() {
```

Comment:
The references to this function has to be removed as well :slightly_smiling_face: